content/v0.0.1/index.md

```diff
@@ -6,3 +6,277 @@ aliases: ["/en/"]
+1. The message is written in past tense since logs are event occurred in the past.
+1. When the same event happens multiple times, the `message` must always be the same and other fields must be used as
+   discriminant.
+1. The `timestamp` field determines when a log entry has been created and contains date+time, it is optional to pass
+   timezone information as well.
+1. An `error` field could be part of a log entry,  
+   and it represents an error (or exception) that cause the log entry to be reported.  
+1. The conventional log specification encourages adding more custom fields to better describe the log entry.
+1. Sensible data, such as customer information ([PII](https://en.wikipedia.org/wiki/Personal_data)),
+   auth token or any other secret must not be added to the entry.
+
+## Specification
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and
+“OPTIONAL” in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+When the same event happens multiple times, the message must always be the same and other fields must be used as
+discriminant.
+
+1. A field for the `message` MUST be provided in each log entry.
+1. The content of a `message` field MUST be a description of an event occurrence.
+1. The content of the `message` field SHOULD be unique within your system.
+1. The content of the `message` field MUST NOT contain variable content and MUST be a static.
+1. The content of the `message` field MUST be written in past tense.
+1. A field for the `timestamp` MUST be provided in each log entry.
+1. The content of the `timestamp` field MUST be a formatted as [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601).
+1. A field for the `severity` MUST be provided in each log entry.
+1. The content of the `severity` field SHOULD be one of ***debug***, ***info***, ***warning*** (or ***warn***),
+   ***error***, ***fatal***.
+1. The content of the `severity` field MUST be all lowercase or uppercase.
+1. A ***debug*** `severity` SHOULD be used only for debugging purposes.
+1. An ***info*** `severity` SHOULD ....
+1. A ***warning*** `severity` (or ***warn***) SHOULD report an event may cause problem to an application but that
+   still does not affect the user directly.
+1. An ***error*** `severity` SHOULD be used when an event needs attention
+   because it affects the user of the application in a disruptive way.
+1. A ***fatal*** `severity` SHOULD flag an entry that caused an application to crash or terminate.
+1. A field for the `error` MAY be provided.
+1. The content of a `error` field MUST be the description regarding an error (exception) that caused the log entry.
+1. A field for the `scope` SHOULD be provided.
+1. The content of a `scope` field MUST be a reference to the creator of the log.
+1. Further entry fields SHOULD be added to a log entry to better.
```

Comment:
I'm not sure I follow this one

Reply:
Hello spells epels lol. My friend I cant read every. bcode and all specifications. Jist cant mabyvreasons. I just trust. Give. If not true will ve put in light. I am nkt. As advancr as u all are. In these fielss. Np lie. Vut who serve. Above all. So kinda how it works. I love to to Talk lovw friends. Hostile students. Now. Peace. Workong. Talk to m. Dm. Me. Poll. . lol. Hope we continue